ngraph/test/frontend/shared/src/op.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <regex>
+#include "../include/op.hpp"
+
+using namespace ngraph;
+using namespace ngraph::frontend;
+
+using TestEngine = test::IE_CPU_Engine;
+
+std::string FrontendOpTest::getTestCaseName(const testing::TestParamInfo<FrontendOpTestParam> &obj) {
+    std::string res = obj.param.m_frontEndName + "_" + obj.param.m_modelName;
+    //res += "I" + joinStrings(obj.param.m_oldInputs) + joinStrings(obj.param.m_newInputs);
```

Comment:
it's fine to have it this way for now, just note that generated 'testCaseName' must be unique for test system.
Combining it only with frontendName and modelName, you'll not be able to run FrontendOptest with same model, but with different inputs